### ISSUE TITLE
"Cannot read property 'useFixedRounds' of undefined" when stopping advanced game

### ISSUE THREAD

**Issue by Peterr-K** [LakeYS/Discord-Trivia-Bot#207]
Hi

on the github version I see this error when I stop the advanced game with !stop

```
/home/user/Discord-Trivia-Bot/triviabot.js:1151
  let useFixedRounds = game[id].config.useFixedRounds;
                                       ^

TypeError: Cannot read property 'useFixedRounds' of undefined
    at Object.Trivia.stopGame (/home/user/Discord-Trivia-Bot/triviabot.js:1151:40)
    at Object.triviaStop (/home/user/Discord-Trivia-Bot/lib/cmd_stop.js:40:16)
    at parseCommand (/home/user/Discord-Trivia-Bot/triviabot.js:1233:14)
    at Object.Trivia.parse (/home/user/Discord-Trivia-Bot/triviabot.js:1510:5)
    at Client.<anonymous> (/home/user/Discord-Trivia-Bot/shard.js:132:19)
```

**Comment by LakeYS** [comment]
Will look into this, although I would advise using the latest release instead in the meantime, if you can. It would also help greatly if you can attach your config (With the token removed, of course) and the options you used with the advanced game.

**Comment by jannyba** [comment]
Hi

I already moved to the web site's version, but the fixed rounds game is very requested. I don't have the config file, but it was set like this below. Only think I was unsure is   "score-multiplier-max": 0, because '0' is the preset value from github and on your we page with instructions is write that '1' is the default doing nothing.
thanks

```
  "token": "123123123",
  "botsfordiscord.com-token": "optionaltokenhere",
  "botlist.space-token": "optionaltokenhere",
  "discordbots.group-token": "optionaltokenhere",
  "discord.bots.gg-token": "optionaltokenhere",
  "discordbots.org-token": "optionaltokenhere",
  "discordbot.world-token": "optionaltokenhere",

  "use-reactions": false,
  "hangman-mode": false,
  "hangman-hints": true,
  "hide-difficulty": false,
  "auto-delete-msgs": true,
  "auto-delete-answers": true,
  "auto-delete-answers-timer": 0,
  "round-length": 20000,
  "round-timeout": 5500,
  "round-end-warnings-disabled": false,
  "rounds-end-after": 2,
  "use-fixed-rounds": true,
  "rounds-fixed-number": 20,
  "disable-score-display": false,
  "score-value": { "easy": 1, "medium": 2, "hard": 3 },
  "score-multiplier-max": 0,
  "command-whitelist": [],
  "accept-first-answer-only": false,
  "reveal-answers": true,

  "prefix": "!",
  "allow-eval": false,
  "shard-count": "auto",
  "disable-admin-commands": false,
  "disable-version-check": false,
  "allow-bots": false,
  "databaseURL": "file://../Otazky/",
  "database-merge": false,
  "database-cache-size": 32,
  "stat-file": "./statistiky.json",
  "embed-color": "006CFF",
  "channel-whitelist": ["791308050714000000"],
  "config-commands-enabled": false,

  "additional-packages": [],
  "additional-packages-root": [],
  "debug-mode": false,
  "debug-log": false,
  "debug-database-flush": false,
  "fallback-mode": false,
  "fallback-silent": false,
  "fallback-exceptions": []
```

**Comment by LakeYS** [comment]
Fixed in fc5701b
